thirdparty/s3.go

```diff
@@ -43,3 +46,28 @@ func PreSign(r RequestParams) (string, error) {
 	urlStr, err := req.Presign(PresignExpireTime)
 	return urlStr, err
 }
+
+func GetHeadObject(r RequestParams) (*s3.HeadObjectOutput, error) {
```

Comment:
Documentation.

Reply:
What do you mean by this? I plan to document the new flag in our wiki, but what kind of documentation would you like to see here?

Reply:
do you mean adding a comment here?

Reply:
Yes, it's considered good practice to document things that are exported, since they're exposing functionality that could be used anywhere. [This blog post](https://go.dev/blog/godoc) talks about good style for Go doc comments, but the gist is to treat writing doc comments like you're writing documentation for the standard library (or any other widely-used Go library).
* Start the comment with the name of the thing you're documenting (i.e. `GetHeadObject`).
* Full sentences and proper grammar are golden.
* Describe expected behavior.
* Describe the expected inputs if they're not obvious.

Reply:
thank you!